anta/cli/nrfu/commands.py

```diff
@@ -63,6 +60,28 @@ def text(ctx: click.Context) -> None:
     exit_with_code(ctx)
 
 
+@click.command()
+@click.pass_context
+@click.option(
+    "--csv-output",
+    type=click.Path(
+        file_okay=True,
+        dir_okay=False,
+        exists=False,
+        writable=True,
+        path_type=pathlib.Path,
+    ),
+    show_envvar=True,
+    required=False,
+    help="Path to save report as a CSV file",
+)
+def csv(ctx: click.Context, csv_output: pathlib.Path) -> None:
+    """ANTA command to check network states with CSV result. It only saves test results and not the output from --group-by option."""
```

Comment:
so right now we don't support group-by on the csv command but only under the `table` command - does it make sense to keep this there? (I added the same comment in the MD report PR).

I guess it is because the target is to support multi CLI later (nrfu table and csv) ?

If so, it would look like the following I suppose: `anta nrfu table --group-by device csv --csv-output=toto.csv`  and so we would need to keep the comment?